reference/webhooks.v3.yml

```diff
@@ -2725,15 +2806,33 @@ components:
       description: |
         Data about the response, including pagination and collection totals.
       properties:
-        offset:
+        total:
           type: integer
-          description: Offset.
-        limit:
+          description: Total number of returned items across all pages.
+        count:
           type: integer
-          description: Total number of items in the result set.
-        total_items:
+          description: Number of items on current page.
+        per_page:
           type: integer
-          description: Total number of items.
+          description: Maximum number of items per page.
+        current_page:
+          type: integer
+          description: Current page number of results.

```

Comment:
This item description is a bit confusing. Does it pertain to the page number or the number of results per current page?

Reply:
Good call, that is confusing. I have just updated to "Current page number"